resources/idlharness.js

```diff
@@ -2240,16 +2240,19 @@ IdlInterface.prototype.test_interface_of = function(desc, obj, exception, expect
             {
                 assert_equals(exception, null, "Unexpected exception when evaluating object");
                 assert_equals(typeof obj, expected_typeof, "wrong typeof object");
+                var fn;
                 if (!member["static"]) {
                     if (!this.is_global() && !member.isUnforgeable) {
                         assert_inherits(obj, member.name);
                     } else {
                         assert_own_property(obj, member.name);
                     }
+                    fn = obj[member.name];
                 }
                 else
                 {
-                    assert_false(member.name in obj);
+                    assert_own_property(obj.constructor, member.name, "constructor does not have property:");
```

Comment:
The prevailing style is to not have a trailing ":", so please omit it.  I agree this does look a bit funny, but if that should be fixed it should be fixed in testharness.js, not by adding a colon to individual asserts in test files.  (I'm thoroughly used to it and don't notice, but maybe you're right it should be fixed, because objectively it does look odd.)